.prettierrc.json

```diff
@@ -1,4 +1,3 @@
 {
-  "trailingComma": "all",
```

Comment:
Default value for `trailingComma` in Prettier [is now `all`](https://prettier.io/blog/2023/07/05/3.0.0.html#change-the-default-value-for-trailingcomma-to-all-11479httpsgithubcomprettierprettierpull11479-by-fiskerhttpsgithubcomfisker-13143httpsgithubcomprettierprettierpull13143-by-sosukesuzukihttpsgithubcomsosukesuzuki).